staging/src/k8s.io/client-go/rest/config.go

```diff
@@ -293,8 +287,7 @@ func InClusterConfig() (*Config, error) {
 	}
 
 	return &Config{
-		// TODO: switch to using cluster DNS.
-		Host:            "https://" + net.JoinHostPort(host, port),
+		Host:            "https://kubernetes.default.svc:443",
```

Comment:
do we want these hard-coded, or do we want the DNS suffix injected (so we can get the FQDN `kubernetes.default.svc.cluster.local`?)